mmv1/products/cloudrun/terraform.yaml

```diff
@@ -332,6 +332,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         skip_docs: true
         vars:
           cloud_run_srv: "cloud-run-srv"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_run_service_probes"
```

Comment:
We'll need to mark this as a beta test per https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/README.md#tests-that-use-a-beta-feature (including annotating the example code)

That's currently blocking generation. It shouldn't, but there's a piece of CI making a bad assumption atm.